admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/constant/UpdateQueryConstants.java

```diff
@@ -7,9 +7,9 @@
  *
  */
 public enum UpdateQueryConstants {
-	BLACKLISTED_WORD_UPDATE_QUERY_WITHOUT_DESCRIPTION(
+	BLOCKLISTED_WORD_UPDATE_QUERY_WITHOUT_DESCRIPTION(
 			"UPDATE BlacklistedWords bw SET bw.updatedBy = :updatedBy , bw.updatedDateTime = :updatedDateTime, bw.word = :word WHERE bw.word = :oldWord and  bw.langCode = :langCode and  (bw.isDeleted is null or bw.isDeleted = false)"),
```

Comment:
Below  in this line https://github.com/mosip/admin-services/pull/276/files#diff-a3242c972b10912364a0328b04893f33e27be4d397b50405f534a5e46f482d4bR28 i see some of the queries changed to BlocklistedWords domain. Then either one of the change is not correct @Govindaraj-035